include/variant_detection/variant_output.hpp

```diff
@@ -10,34 +10,46 @@
 
 /*! \brief Detects genomic variants from junction clusters and prints them to output stream in VCF format.
  *
- * \param[in]       clusters    - input junction clusters
- * \param[in]       args        - command line arguments:\n
- *                                **args.min_var_length** - minimum length of variants to detect (expected to be non-negative) - *default: 30 bp* \n
- *                                **args.max_var_length** - maximum length of variants to detect (expected to be non-negative) - *default: 1,000,000 bp*\n
- *                                **args.max_tol_inserted_length** - longest tolerated inserted sequence at non-INS SV types (expected to be non-negative) - *default: 5 bp*
- * \param[in, out]  out_stream  - output stream
+ * \param[in]       references_lengths  - a vector of \@SQ header tags tuples: SN and LN
```

Comment:
You already changed the description of this parameter in `variant_detection.hpp`. Could you also do it here? :)